packages/paste-core/components/tooltip/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@twilio-paste/tooltip",
+  "version": "0.0.0",
+  "category": "interaction",
+  "status": "alpha",
+  "description": "The Tooltip is a popup that displays clarifying text related to an element that has been focused or hovered on.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -cw --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^2.5.4",
+    "@twilio-paste/button": "^1.2.11",
```

Comment:
This package isn't required. Stories work differently, so we don't need to list story packages in the package.json